contracts/WalletSimple.sol

```diff
@@ -282,14 +268,15 @@ contract WalletSimple is IERC721Receiver, ERC1155Receiver {
   function batchTransfer(
     address[] calldata recipients,
     uint256[] calldata values
-  ) internal {
+  ) private {
     for (uint256 i = 0; i < recipients.length; i++) {
+      require(recipients[i] != address(0), 'Invalid recipient address');
```

Comment:
Do we need this check added here, address zero might help you in confirming that your contract is handling uninitialised data correctly, But in this case since we are already checking recipients.length and only passing in if the value is there, this check seems redundant

Reply:
Removed the zero address validations